### ISSUE TITLE
Gitea fails to create or migrate avatar directory correctly.

### ISSUE THREAD

**Issue by Starz0r** [go-gitea/gitea#1057]
- Gitea version (or commit ref):
1.0.0+334-g80f900e
- Git version:
2.7.4
- Operating system:
Linux Mint 18.1
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
```
10:04:40 [...itea/routers/init.go:54 GlobalInit()] [E] Failed to initialize ORM engine: migrate: do migrate: open /var/www/git/data/avatars: no such file or directory
```
## Description
When updating from the previous master branch I noticed that Gitea would not start, When checking the logs it seems it tries to migrate to solve this issue, but doesn't actually do anything. Creating the /avatars/ directory in the /data/ directory solves the issue of Gitea not being able to start.

**Comment by strk** [comment]
This is fixed by merging #1056

**Comment by strk** [comment]
@lunny

**Comment by lunny** [comment]
resolved by #1056
